spec/rubocop/cop/layout/space_in_lambda_literal_spec.rb

```diff
@@ -9,7 +9,7 @@
     it 'registers an offense for no space between -> and (' do
       expect_offense(<<-RUBY.strip_indent)
         a = ->(b, c) { b + c }
-            ^^^^^^^^^^^^^^^^^^ Use a space between `->` and opening brace in lambda literals
+            ^^^^^^^^^^^^^^^^^^ Use a space between `->` and opening parenthesis in lambda literals
```

Comment:
I think that's the real problem - the cop highlights way too much and it's not clear what it's referring to. @koic often fixes problems like this one.

Reply:
Ideally we should highlight just `->(b, c)` in this example.